test/parallel/test-console-async-write-error.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+require('../common');
+const { Console } = require('console');
+const { Writable } = require('stream');
+const assert = require('assert');
+
+const out = new Writable({
+  write(chunk, enc, callback) {
```

Comment:
@jasnell Done, good point. (Sorry for being late, must have overlooked your replies.)